demos/run/main.py

```diff
@@ -0,0 +1,276 @@
+import random
```

Comment:
Remove this game from the this commit, doesn't fit within the scope of your branch